iree/samples/variables_and_state/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 The IREE Authors
```

Comment:
mmm this file is nice

Reply:
Kudos to @marbre for setting up `iree-template-cpp` :)

I referenced the basic CMake target setup at https://github.com/iml130/iree-template-cpp/blob/92f8b06f2c6956e37397ff835796fdbbcb84cb7d/simple_embedding/CMakeLists.txt#L72-L98 when writing this

although looking at this more closely, I think I can also remove the `target_include_directories`

Reply:
Glad if `iree-template-cpp` helped :) And yes, you can drop `target_include_directories` as you don't include a generated header file.